x/claim/keeper/msg_server_add_claim.go

```diff
@@ -0,0 +1,17 @@
+//go:build !testnet
+
+package keeper
+
+import (
+	"context"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+
+	"github.com/arkeonetwork/arkeo/x/claim/types"
+)
+
+func (k msgServer) AddClaim(goCtx context.Context, msg *types.MsgAddClaim) (*types.MsgAddClaimResponse, error) {
+	ctx := sdk.UnwrapSDKContext(goCtx)
+	_ = ctx
```

Comment:
should we throw a descriptive error here instead of just silently doing nothing? may help us debug if we ever have the wrong build flag somewhere.

Reply:
yup, make sense, return an error , so it will cause the transaction to fail